### ISSUE TITLE
Could not find a valid mapping for path "/auth/twitter/callback"

### ISSUE THREAD

**Issue by sea-grass** [cantino/huginn#826]
I created an app in Twitter and set the callback to http://my-server:3000/auth/twitter/callback and copied my API KEY and SECRET in my .env file. When I go to /services and Authenticate with Twitter, I enter my credentials and am redirected to a RuntimeError page with the following error message:  

`Could not find a valid mapping for path "/auth/twitter/callback"`

**Comment by cantino** [comment]
Hey @sea-grass, welcome to the project!

Would you be able to share how you put the key and secret in your .env file?  Just replace some of the key and secret with xxxxxxx to keep it secure.

**Comment by sea-grass** [comment]
Thanks @cantino! Sure thing.

The key and secret in my .env file look like this:

```
TWITTER_OAUTH_KEY=bgxxxxxxxxxxxxxxxxxxxx3LN
TWITTER_OAUTH_SECRET=dlxxxxxxxxxxxxxxxxxxxx7BiofJoh
```

**Comment by cantino** [comment]
Very odd, that looks fine.  Things that come to mind:

1. Are you using `bundle exec foreman start`?
1. Have you commented out anything in the Gemfile?
1. Did you follow [these instructions](https://github.com/cantino/huginn/wiki/Configuring-OAuth-applications)?
1. If you run `bundle exec rails c` and then type `OmniAuth::Strategies::Twitter` at the console, does it give an error? (Exit with `exit` afterwards.)

**Comment by sea-grass** [comment]
Thanks for the directions. I am new to ruby so I don't know where to start debugging it.

1. Yes
2. No
3. Yes
4. I ran `bundle exec rails c` but it kept hanging while it was loading, so following this [answer](http://stackoverflow.com/a/25172613) on SO, I was able to enter the ruby console by running `bin/spring stop` first.

This is my interaction with the console:
```ruby
[seagrass@sea-server huginn]$ bundle exec rails c
Loading development environment (Rails 4.2.1)
irb: warn: can't alias context from irb_context.
irb(main):001:0> OmniAuth::Strategies::Twitter
=> OmniAuth::Strategies::Twitter
irb(main):002:0> exit
```
No error as far as I can tell (minus the `irb: warn: can't alias context from irb_context` warning).
